common/src/decompress.rs

```diff
@@ -27,6 +28,7 @@ pub enum Error {
 /// **NOTE:** Depending on the size of the payload, this method might take some time. In an async
 /// context, it might be necessary to run this as a blocking function, or use [`decompress_async`]
 /// instead.
+#[instrument(skip(bytes), fields(bytes_len=bytes.len()), err(level=tracing::Level::INFO))]
```

Comment:
you explained this in the design doc but wanted to explicitly ask - the `err(level=tracing::Level::INFO)` is saying log errors from this func to info ... though this is not stopping halt of the application itself or wrapping up error mode behaviour.. right ? I could probably ask this question better!

Reply:
Correct, the only affect of this that the error gets logged as "info", rather than "error". Nothing else changes. The idea behind it being, that at this point, the error does not affect the application itself. So that doesn't warrant an "error log".

Reply:
Not sure if there's something to add to the doc.